### DISCUSSION TITLE
setup.py

### DISCUSSION THREAD

**Discussion by aiotter** [sizumita/discord-ext-ui#6] [General]
ここなにしてるん
https://github.com/sizumita/discord-ext-ui/blob/bc092e50db50c3b8112d40b7bca3a3bf81f6f704/setup.py#L11-L27

**Comment by sizumita** [comment]
これはアルファ版とかベータ版を落としてくる時に、バージョンに差異がないと新しいやつを入れてくれないので、gitのコミットから差異を作っているコード（なはず）

**Reply by peacock0803sz** [reply]
どっかのコピペコードで意図理解してないなら使わない方がいいのでは

**Reply by sizumita** [reply]
なかったら自分でgithubから落とすときにうまくいかなかったのでつけた感じです
